### PR TITLE
Don't ping people in PRs

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

**Comment by matklad** [comment]
That's because PR description gets folded in the commit message by bors, right? Let's clarify that: in generall, mentioning in PR description should be fine. Might also make sense to open an issue on bors-ng repo to santilize mentyions

**Comment by lnicola** [comment]
Yeah

File: docs/dev/README.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ We don't have specific rules around git history hygiene.
 Maintaining clean git history is encouraged, but not enforced.
 We use rebase workflow, it's OK to rewrite history during PR review process.
 
-Avoid @mentioning people in commit messages, as such messages create a lot of duplicate notification traffic during rebases.
+Avoid @mentioning people in commit messages and pull request descriptions, as such messages create a lot of duplicate notification traffic during rebases.
```
Comment (matklad) [inline docs/dev/README.md:317]:
```suggestion
Avoid @mentioning people in commit messages and pull request descriptions (they are added to commit message by bors), as such messages create a lot of duplicate notification traffic during rebases.
```

**Comment by lnicola** [comment]
Sorry, I didn't get a chance to make that change.

**Comment by matklad** [comment]
That's ok, I just edited the PR, will open the issue now!

**Pull request merged**